TODO

```diff
@@ -24,6 +24,12 @@ Janitorial Clean-ups:
 
 Features:
 
+* rework mount.c and swap.c to follow proper state enumeration/deserialization
+  semantics, like we do for device.c now
+
+* When reloading configuration PID 1 should reset all its properties to the
+  original defaults before calling parse_config()
+
```

Comment:
That's right I noticed that too. But I wouldn't call that a feature ;)

Reply:
Yeah, it's buggy behaviour that we should fix. Hence I added it to the TODO list.